SQL tutorial fixes (#294)
hazelcast/hz-docs: docs/modules/sql/pages/get-started-sql.adoc

+
[source,sql]
----
CREATE MAPPING cities (
__key INT, 
countries VARCHAR, 
cities VARCHAR)
type IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat');
----

> . Add some countries and their capital cities to the map.

Reviewer: ```suggestion
. Add some cities and countries to the map.
```

No longer just captials.

|           7|Brazil              |Sao Paulo           |
|           8|Brazil              |Rio de Janeiro      |
|           5|Turkey              |Istanbul            |
|           4|United States       |Los Angeles         |
|           3|United States       |New York            |
+------------+--------------------+--------------------+

----

. Query only the countries by filtering on the `countries` column.
+
[source,sql]
----
SELECT countries FROM cities;
----
+
```
+--------------------+
|countries           |
+--------------------+
|United Kingdom      |
|Turkey              |
|United Kingdom      |
|Brazil              |
|Brazil              |
|Turkey              |
|United States       |
|United States       |
+--------------------+
```

. Query only the cities by filtering on the `cities` column.
+
[source,sql]
----
SELECT cities FROM cities;
----
+
```
+--------------------+
|cities              |
+--------------------+
|Manchester          |
|Ankara              |
|London              |
|Sao Paulo           |
|Rio de Janeiro      |
|Istanbul            |
|Los Angeles         |
|New York            |
+--------------------+
```

> . Change the output to display cities first. Change the column labels. Place the city names in alphabetical order.

Reviewer: ```suggestion
. Change the output to display cities first in alphabetical order. The `AS` command renames the columns to the given aliases.
```

These three short sentences in a row were choppy and a bit jarring.

+
NOTE: This clause does not rename the column in the table.
+
[source,sql]
----
SELECT cities AS City, countries AS Country
FROM cities
ORDER BY cities;
----
+
```
+--------------------+--------------------+
|City                |Country             |
+--------------------+--------------------+
|Ankara              |Turkey              |
|Istanbul            |Turkey              |
|London              |United Kingdom      |
|Los Angeles         |United States       |
|Manchester          |United Kingdom      |
|New York            |United States       |
|Rio de Janeiro      |Brazil              |
|Sao Paulo           |Brazil              |
+--------------------+--------------------+

```

. Use a filter to display only countries where the name of the city is at least 11 characters long.
+
[source,sql]
----
SELECT countries FROM cities WHERE LENGTH(cities) >= 11;
----
+
```
+--------------------+
|countries           |
+--------------------+
|Brazil              |
|United States       |
+--------------------+
```

> . Use another filter to display only city names beginning with the letter L where the length is > 6.

Reviewer: ```suggestion
. Use another filter to display only cities beginning with the letter 'L' where the length is greater than 6.
```

+
[source,sql]
----
SELECT cities AS City
FROM cities

> WHERE cities LIKE 'L%' AND LENGTH(cities) >6;

Reviewer: ```suggestion
WHERE cities LIKE 'L%' AND LENGTH(cities) > 6;
```

(2, 'Manchester', 2730076),
(3, 'New York', 8622357),
(4, 'Los Angeles', 4085014),
(5, 'Sao Paulo', 12396372),
(6, 'Rio de Janeiro', 6775561),
(7, 'Istanbul', 14804116),
(8, 'Ankara', 3517182);
----

. Use the `JOIN` clause to merge results from the `cities` and `population2020` tables so you can see which countries had the most populated captial cities in 2020.
+
[source,sql]
---- 
SELECT cities.countries AS country, cities.cities AS city, population2020.population AS population
FROM cities
JOIN population2020
ON cities.cities = population2020.cities;
----
+
```
+--------------------+--------------------+------------+
|country             |city                |  population|
+--------------------+--------------------+------------+
|United Kingdom      |Manchester          |     2730076|
|Turkey              |Ankara              |     3517182|
|United Kingdom      |London              |     9304016|
|Brazil              |Sao Paulo           |    12396372|
|Brazil              |Rio de Janeiro      |     6775561|
|Turkey              |Istanbul            |    14804116|
|United States       |Los Angeles         |     4085014|
|United States       |New York            |     8622357|

> +--------------------+--------------------+------------+```

Reviewer: The backticks here need to be on a new line. This format is breaking the layout.

Author: FIXED!

> . Use the `ORDER BY` clause to order the results based on population, largest first.

Reviewer: ```suggestion
. Use the `ORDER BY` clause to order the results by population, largest first.
```

+--------------------+
|    total_population|
+--------------------+
|            62234694|
+--------------------+

```

. Filter for cities that had a population of more than 5,000,000 in 2020.
+
[source,sql]
----
SELECT population2020.cities AS large_cities
FROM population2020
WHERE population2020.population > 5000000;
----
+
```
+--------------------+
|large_cities        |
+--------------------+
|New York            |
|Rio de Janeiro      |
|London              |
|Istanbul            |
|Sao Paulo           |
+--------------------+
```

> . Display the names of countries and the sum of the city populations. Order by population in ascending order.

Reviewer: ```suggestion

. Display the names of countries and the sum of the city populations. Order by population in ascending order.
```

+
[source,sql]
----

> SELECT cities.countries AS country, sum(population2020.population) AS total_population

Reviewer: ```suggestion
SELECT cities.countries AS country, SUM(population2020.population) AS total_population
```

FROM cities
JOIN population2020
ON cities.cities = population2020.cities
GROUP BY cities.countries
ORDER by sum(population2020.population);
----
+
```
+--------------------+--------------------+
|country             |    total_population|
+--------------------+--------------------+
|United Kingdom      |            12034092|
|United States       |            12707371|
|Turkey              |            18321298|
|Brazil              |            19171933|
+--------------------+--------------------+
```

> +

Reviewer: ```suggestion

```
